.github/workflows/v1.deployDotNetCoreAppToAzureWebApp.yml

```diff
@@ -1,5 +1,11 @@
+name: V1_BuildAndDeploy_AspNetCoreApp_ToAzureWebApp
+
 on:
   push:
+    paths:
+      - sampleApps/dotNetCore/**
+      - '!.github/workflows/**'
+      - '.github/workflows/v1.deployDotNetCoreAppToAzureWebApp.yml'
```

Comment:
craziness: https://github.community/t5/GitHub-Actions/Trigger-a-workflow-on-change-to-the-yml-file-itself/m-p/49332